pyfr/shapes.py

```diff
@@ -162,6 +162,11 @@ def upts(self):
         rname = self.cfg.get(f'solver-elements-{self.name}', 'soln-pts')
         return get_quadrule(self.name, rname, self.nupts).pts
 
+    @cached_property
+    def upts_wts(self):
+        rname = self.cfg.get(f'solver-elements-{self.name}', 'soln-pts')
```

Comment:
Can we use `_get_qrule` here to make it a one liner?

Reply:
I think `_get_qrule` would return the qpts weights if `quad-pts` or `quad-deg` are specified in the element section which isn't what we want